docs/topic_guides/customising_comparisons.ipynb

```diff
@@ -213,7 +213,90 @@
+    "\n",
+    "The key difference is that `ComparisonTemplateLibrary` contains functions to generate a 'best practice' `Comparison` based on the type of data in a given column. \n",
+    "\n",
+    "The following provides an example of using the ComparisonTemplateLibrary for DuckDB."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 45,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Comparison 'Exact match vs. Dates within Levenshtein threshold 2 vs. Dates within the following thresholds Month(s): 1, Year(s): 1 vs. anything else' of \"date_of_birth\".\n",
+      "Similarity is assessed using the following ComparisonLevels:\n",
+      "    - 'Null' with SQL rule: \"date_of_birth_l\" IS NULL OR \"date_of_birth_r\" IS NULL\n",
+      "    - 'Matching and 1st Jan' with SQL rule: date_of_birth_l = date_of_birth_r AND\n",
+      "                                    substr(date_of_birth_l, 6, 5) = '01-01'\n",
+      "    - 'Exact match' with SQL rule: \"date_of_birth_l\" = \"date_of_birth_r\"\n",
+      "    - 'Levenshtein <= 2' with SQL rule: levenshtein(\"date_of_birth_l\", \"date_of_birth_r\") <= 2\n",
+      "    - 'Within 1 month' with SQL rule: \n",
+      "        abs(date_diff('month', \"date_of_birth_l\", \"date_of_birth_r\")) <= 1\n",
+      "    \n",
+      "    - 'Within 1 year' with SQL rule: \n",
+      "        abs(date_diff('year', \"date_of_birth_l\", \"date_of_birth_r\")) <= 1\n",
+      "    \n",
+      "    - 'All other comparisons' with SQL rule: ELSE\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "from splink.duckdb.duckdb_comparison_template_library import (\n",
+    "    date_comparison\n",
+    ")\n",
+    "\n",
+    "date_of_birth_comparison = date_comparison(\"date_of_birth\")\n",
+    "print(date_of_birth_comparison.human_readable_description)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "These `Comparisons` can be specified in a data linking model as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 46,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "settings = {\n",
+    "    \"link_type\": \"dedupe_only\",\n",
+    "    \"blocking_rules_to_generate_predictions\": [\n",
+    "        \"l.first_name = r.first_name\",\n",
+    "        \"l.surname = r.surname\",\n",
+    "    ],\n",
+    "    \"comparisons\": [\n",
+    "        exact_match(\"first_name\"),\n",
+    "        date_comparison(\"dob\"),\n",
+    "\n",
+    "    ]\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can customise a `ComparisonTemplate` by chosing your own values for the [function parameters](../comparison_template_library.md), but for anything more bespoke you will want to construct a `Comparison` with `ComparisonLevels` or providing the spec as a dictionary."
```

Comment:
Couple of typos: chosing -> choosing & providing -> provide

Reply:
Good spot - fixed